Health/Core/NavigationBar/HealthNavigationBar/HealthNavigationBar.swift

```diff
@@ -151,7 +151,7 @@ final class HealthNavigationBar: CoreView {
         config.title = item.title
         config.image = item.image
         config.preferredSymbolConfigurationForImage = defaultTrailingBarButtonItemSymbolConfiguration
-            .applying(preferredTrailingBarButtonItemSymbolConfiguration ?? defaultTitleImageSymbolConfiguration)
+            .applying(preferredTrailingBarButtonItemSymbolConfiguration ?? defaultTrailingBarButtonItemSymbolConfiguration)
```

Comment:
@rlarjsdn3 
로직이 뭔가 이상한거 같아서 아래가 맞지 않을지 한 번 봐주세요!
```suggestion
        config.preferredSymbolConfigurationForImage = preferredTrailingBarButtonItemSymbolConfiguration ?? defaultTrailingBarButtonItemSymbolConfiguration
```